x-pack/plugins/triggers_actions_ui/kibana.json

```diff
@@ -7,9 +7,9 @@
   "version": "kibana",
   "server": true,
   "ui": true,
-  "optionalPlugins": ["alerting", "cloud", "features", "home", "spaces"],
-  "requiredPlugins": ["management", "charts", "data", "kibanaReact", "kibanaUtils", "savedObjects", "unifiedSearch", "dataViews"],
+  "optionalPlugins": ["cloud", "features", "home", "spaces"],
+  "requiredPlugins": ["management", "charts", "data", "kibanaReact", "kibanaUtils", "savedObjects", "unifiedSearch", "dataViews", "alerting", "actions"],
   "configPath": ["xpack", "trigger_actions_ui"],
   "extraPublicDirs": ["public/common", "public/common/constants"],
-  "requiredBundles": ["alerting", "esUiShared", "kibanaReact", "kibanaUtils"]
+  "requiredBundles": ["alerting", "esUiShared", "kibanaReact", "kibanaUtils", "actions"]
```

Comment:
I figured we need to add the actions bundle here, since it's now referencing code there.  Though I think it worked at dev time without it - or maybe I was imagining that.